app/models/application_setting.rb

```diff
@@ -64,5 +80,16 @@ def load_defaults(var_name)
     def unscoped
       Setting.unscoped
     end
+
+    def save_default
+      @keys.each do |key|
+        unless Setting.exists?(var: key)
+          s = Setting.new
```

Comment:
__[Reek]__

[UncommunicativeVariableName] ApplicationSetting#save_default has the variable name 's'


<!-- Comment by [Sider](https://sider.review) -->